classes/submission/PKPSubmissionFileDAO.inc.php

```diff
@@ -462,6 +462,29 @@ function assignRevisionToReviewRound($fileId, $revision, $reviewRound) {
 		);
 	}
 
+	/**
+	 * Update Submission File with id to set assocType and assocId.
+	 * Used to native import process
+	 * @param $fileId int
+	 * @param $assocType int ASSOC_TYPE_...
+	 * @param $assocId int
+	 */
+	function assignAssocTypeAndIdToFile($fileId, $assocType, $assocId) {
+		// Update the file in the database.
+		$this->update(
+			'UPDATE submission_files
```

Comment:
Beware that this will set the assoc information for all revisions. Is that intentional?